packages/components/molecules/f-mega-modal/src/components/MegaModal.vue

```diff
@@ -367,32 +378,39 @@ export default {
     }
 }
 
-    .c-megaModal-content--visible {
-        display: block;
-    }
+.c-megaModal-content--visible {
+    display: block;
+}
 
-    .c-megaModal-content--narrow {
-        @include media('>=narrowMid') {
-            max-width: 450px;
-        }
+.c-megaModal-content--narrow {
+    @include media('>=narrowMid') {
+        max-width: 450px;
     }
+}
 
-    .c-megaModal-content--wide {
-        max-width: 1080px;
+.c-megaModal-content--wide {
+    max-width: 1080px;
 
-        @include media('<huge') {
-            max-width: 75%;
-        }
+    @include media('<huge') {
+        max-width: 75%;
+    }
 
-        @include media('<mid') {
-            width: 100%;
-            max-width: 100%;
-        }
+    @include media('<mid') {
+        width: 100%;
+        max-width: 100%;
     }
+}
 
-    .c-megaModal-content--flush {
-        padding: 0;
+.c-megaModal-content--flush {
+    padding-top: 0;
```

Comment:
This change will break the modals in menu (and potentially in other projects) where we need to remove all spacing from the modal so we can handle it ourselves. Was there any particular reason for only applying to the top and bottom?

Reply:
@jamieomaguire Bump on this one